tests/test_flows.sh

```diff
@@ -200,6 +201,23 @@ test_client_capability_query() {
     return $check_error
 }
 
+test_link_metric_query() {
+    status "test link metric query"
+    check_error=0
+    check send_CAPI_1905 ${GATEWAY} $mac_agent1 0x0005 "tlv_type,0x08,tlv_length,0x0002,tlv_value,0x00 0x02"
+    sleep 1
+    
+    dbg "Confirming link metric query has been received on agent"
+    check_log ${REPEATER1} agent "Received LINK_METRIC_QUERY_MESSAGE"
+    
+    dbg "Confirming link metric response has been received on controller"
+    check_log ${GATEWAY} controller "Received LINK_METRIC_RESPONSE_MESSAGE"
+    check_log ${GATEWAY} controller "Received TLV_TRANSMITTER_LINK_METRIC"
+    check_log ${GATEWAY} controller "Received TLV_RECEIVER_LINK_METRIC"
```

Comment:
This should of course be converted to Python.

Reply:
Done